src/pages/blog/[...page].astro

```diff
@@ -18,7 +18,7 @@ const { page } = Astro.props;
 ---
 
 <Base meta_title="Blog">
-  <h1 class="text-2xl md:text-3xl font-bold mb-8">Blog</h1>
+  <h2 class="text-2xl md:text-3xl font-bold mb-8">Blog</h2>
```

Comment:
What is the proposal for changing h1 to h2?

Reply:
There should only be one H1 header per page, in this case, along with the hero section title and the titles of the pages: Blog, Tags, Categories. Having the H1 header duplicated in these pages would occur.